hubstorage/client.py

```diff
@@ -88,10 +88,12 @@ def request(self, is_idempotent=False, **kwargs):
         def invoke_request():
             r = self.session.request(**kwargs)
 
-            if not r.ok:
+            try:
+                r.raise_for_status()
+                return r
+            except RequestException:
```

Comment:
raise_for_status() raises only `HTTPError` exceptions, no reason to use parent `RequestException` class